users/config.rst

```diff
@@ -124,6 +124,7 @@ The following shows an example of the default configuration file (IDs will diffe
             <kcpReceiveWindowSize>128</kcpReceiveWindowSize>
             <defaultFolderPath>~</defaultFolderPath>
             <minHomeDiskFreePct>0</minHomeDiskFreePct>
+            <setLowPrio>true</setLowPrio>
```

Comment:
So is it prio or priority? It should be priority everywhere I think.

Reply:
Whoops